frontend/gwms_renew_proxies.py

```diff
@@ -42,12 +42,13 @@ def timeleft(self):
 class VO(object):
     """Class for holding information related to VOMS attributes
     """
-    def __init__(self, vo, command, cert=None, key=None):
+    def __init__(self, vo, fqan, cert=None, key=None):
+        if not fqan.startswith('/%s/' % vo):
+            raise ValueError('FQAN (%s) does not begin with specified VO (%s). Verify %s.' % (fqan, vo, CONFIG))
         self.name = vo
-        command = command.lstrip('/')
-        self.fqan = '/%s/%s' % (vo, command)
+        self.fqan = fqan
         # intended argument for -voms option "vo:command" format, see voms-proxy-init man page
-        self.voms = '%s:/%s' % (vo, command)
+        self.voms = fqan.replace('/%s' % vo, vo + ':')
```

Comment:
Is this really what we want here?  Does it replace all instances?  Or do we just want the first one?